packages/cli/templates/webcomponents/igc-ts/projects/side-nav/index.ts

```diff
@@ -0,0 +1,44 @@
+import { ControlExtraConfiguration, defaultDelimiters, ProjectTemplate, Util } from "@igniteui/cli-core";
```

Comment:
Same logic applies to the Project logic itself, which is again (sans a default title) the same - move to a base project and inherit the shared logic